brainscore_vision/models/cvt_cvt_21_384_in1k_4/__init__.py

```diff
@@ -0,0 +1,9 @@
+from brainscore_vision import model_registry
+from brainscore_vision.model_helpers.brain_transformation import ModelCommitment
+from .model import get_model, get_layers
+
+
+model_registry['cvt_cvt-13-384-in1k_4'] = \
+    lambda: ModelCommitment(identifier='cvt_cvt-13-384-in1k_4',
+                            activations_model=get_model('cvt_cvt-13-384-in1k_4'),
+                            layers=get_layers('cvt_cvt-13-384-in1k_4'))
```

Comment:
It appears that your `__init__.py` file for your `cvt_cvt_21_384_in1k_4` and `cvt_cvt_13_384_in_1k_4` are swapped and creating inconsistencies in the model_identifier for everything else in the respective directories.

Should be an easy fix and once that's done, I think this should pass.